corehq/pillows/mappings/xform_mapping.py

```diff
@@ -121,6 +123,9 @@
     }
 }
 
+if settings.ES_XFORM_DISABLE_ALL:
+    XFORM_MAPPING["_all"] = {"enabled": False}
```

Comment:
I think it makes sense to follow a similar approach to the `meta` which puts this logic in the `ElasticsearchIndexInfo`: https://github.com/dimagi/commcare-hq/blob/709101a81aa349e7ad473630786924bcf0fecd8f/corehq/ex-submodules/pillowtop/es_utils.py#L86

Reply:
I thought it's better to keep all the mapping related things here in one file instead of customizing it in the other file. Also, the `meta` method customizes index settings and not the mapping.

Reply:
yes, I was suggesting doing the same for metadata as we do for meta to keep all the env specific adjustments in one place rather than have them spread about.

Reply:
But this way, all the mapping would be in once place, no (if one looks at a mapping file one would know all customizations possible, without this one would assume that the mapping defined here is always the final)? Though, you are right about customizations based on settings is in two places.

Reply:
both have merits, fine to leave as is for now

Reply:
Cool